AMAZON-INSTRUCTIONS.md

```diff
@@ -18,12 +18,56 @@ This release adds pre-release support for Amazon store.
+purchases.Setup(builder.Build());
 ```
-       var builder = PurchasesConfiguration.Builder.Init("amazon_specific_api_key")
-            .SetUseAmazon(true);
-        purchases.Setup(builder.Build());
-```
 
-Due to some limitations, RevenueCat will only validate purchases made in production or in Live App Testing and won't validate purchases made with the Amazon App Tester. You can read more about the different testing environments in [our Amazon (Beta) docs](https://docs.revenuecat.com/docs/amazon-store-beta#sandbox-testing).
+- Due to technical limitations, RevenueCat will only validate purchases made in production or in Live App Testing and won't validate purchases made with the Amazon App Tester. You can read more about the different testing environments in [our Amazon (Beta) docs](https://docs.revenuecat.com/docs/amazon-store-beta#sandbox-testing).
+
+- Due to technical limitations with the Amazon SDK, `SyncPurchases` doesn't work when on Amazon observer mode. In order to sync purchases with RevenueCat you have to send the current active subscriptions when Unity IAP initializes and after every successful purchase. For example:
+
+ ```c#
+public void OnInitialized(IStoreController controller, IExtensionProvider extensions)
+{
+    m_StoreController = controller;
+    storeExtensionProvider = extensions;
+    var purchases = GetComponent<Purchases>();
+    purchases.SetDebugLogsEnabled(true);
+    foreach (Product product in controller.products.all)
+    {
+        if (product.hasReceipt) {
+            var amazonExtensions = storeExtensionProvider.GetExtension<IAmazonExtensions>();
+            var userId = amazonExtensions.amazonUserId;
+            purchases.SyncObserverModeAmazonPurchase( 
+                product.definition.id,
+                product.transactionID,
+                userId,
+                product.metadata.isoCurrencyCode,
+                Decimal.ToDouble(product.metadata.localizedPrice)
+            );
+        }
+    }
+}
+
+public PurchaseProcessingResult ProcessPurchase(PurchaseEventArgs e)
+{
+    var purchases = GetComponent<Purchases>();
+    
+    var amazonExtensions = storeExtensionProvider.GetExtension<IAmazonExtensions>();
+    var userId = amazonExtensions.amazonUserId;
+    purchases.SyncObserverModeAmazonPurchase(
+        e.purchasedProduct.definition.id,
+        e.purchasedProduct.transactionID,
+        userId,
+        e.purchasedProduct.metadata.isoCurrencyCode,
+        Decimal.ToDouble(e.purchasedProduct.metadata.localizedPrice)
+    );
+    return PurchaseProcessingResult.Complete;
+}
+ ```
+
+ - Perform a Resolve using the editor option `Assets/External Dependency Manager/Android Resolver/Resolve` menu. This will add the right dependencies to the `mainTemplate.gradle`.
 
-RevenueCat doesn't support Amazon in observer mode yet
+ - In observer mode, adding the Amazon in-app-purchasing library jar is not necessary since it will be added by Unity IAP when targeting the Amazon Store
```

Comment:
I would remove this step too. In the early versions of our Amazon support, the instructions included a reference to adding the amazon jar, but that's not needed anymore because the Amazon dependency is downloaded from Maven.